packages/@react-aria/menu/docs/useMenuTrigger.mdx

```diff
@@ -120,7 +120,7 @@ function MenuButton(props) {
       {state.isOpen &&
         <MenuPopup
           {...props}
-          domProps={menuProps}
+          {...menuProps}
```

Comment:
close, but I think we'd want to leave this as domProps so that props get passed to MenuPopup hooks, but domProps do not
I think if you just convert from label to aria-label, that should be sufficient
```
{...props}
domProps={menuProps}
aria-label={props.label}
```

Reply:
Thanks @snowystinger , I've updated my branch.